spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/snippet/StandardTableSnippet.java

```diff
@@ -109,16 +101,30 @@ protected void enrichModel(Map<String, Object> model, HandlerMethod handlerMetho
         return model;
     }
 
-    private String stringOrEmpty(Object value) {
+    private String toString(Object value) {
         if (value != null) {
-            return value.toString();
+            return trimToEmpty(value.toString());
         } else {
             return "";
         }
     }
 
-    private String determineLineBreak(Operation operation) {
-        return getTemplateFormat(operation).getId().equals(TemplateFormats.asciidoctor().getId())
-                ? LINE_BREAK_ASCIIDOC : LINE_BREAK_MARKDOWN;
+    private String joinAndFormat(String lineBreak, String description, List<String> constraints) {
+        StringBuilder str = new StringBuilder(description);
+        if (str.length() > 0 && str.charAt(str.length() - 1) != '.') {
```

Comment:
This can be shortened to `if (!description.endsWith("."))`. It also improves readability.